### ISSUE TITLE
Can't open cell workbench in multiplayer fabric servers

### ISSUE THREAD

**Issue by MissPotato** [AppliedEnergistics/Applied-Energistics-2#4878]
**Describe the bug**
When right clicking on the cell workbench the GUI doesn't open. It throws an error in console instead. (Linked below)

**To Reproduce**
Craft cell workbench, right click it to open.

**Expected behavior**
Gui to open.

**Additional context**
https://pastebin.com/HbiHpzMs
We also have [Mixin Trace ](https://github.com/comp500/mixintrace/releases/tag/1.0.0 ) added to the pack due to another bug so the log should have more info
Doesn't cause a crash and is server side only so force crashing doesn't really help but my entire client log is: https://paste.ee/p/A3OhU

**Environment**
Pack: All of Fabric 3, which uses AE2 V8.1.0-alpha.13
OpenJDK Runtime Environment build 1.8.0_272-b10

**Comment by AlexIIL** [comment]
This is caused by a recent LBA update which validates that the `slot` passed to `SingleItemSlotView` is within the bounds of the inventory. However this doesn't work properly with the cell workbench, since it sets up the slots in it's super class before the workbench inventory is set in the subclass [CellWorkbenchContainer.java#L71-L72](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/fabric/master/src/main/java/appeng/container/implementations/CellWorkbenchContainer.java#L71-L72).

~~Personally I'd recommend overriding `UpgradableContainer.getUpgradeable()` to make it cast the return value of the super class to `CellWorkbenchBlockEntity`, since that way it's always usable. However if you don't want to do that I can revert that additional validation, since it's not technically necessary.~~ *After some investigation it looks like you always create 24 slot views, which will break if I don't revert. So, I'll just revert that change to LBA.*

**Comment by MissPotato** [comment]
@AlexIIL I'm a tad confused, the version of AE2 in All of Fabric 3 is using LibLockAttributes-Core0.8.0.jar according to the fabric.mod.json.  How is the change in 0.8.5 causing this? Might need to dumb it down a little for me as I'm not very familiar with java.

**Comment by AlexIIL** [comment]
Sorry, I should have specified that most of my message was aimed towards AE2's developers.

Fabric tries to load the newest version of mods that it can find - and it looks like the server has added a newer version of LibBlockAttributes, which is causing the crash. (Your client log shows that you're loading LBA 0.8.4, although I have no idea where from as fabric doesn't log that ATM).

**Comment by MissPotato** [comment]
Ah that makes sense, if you want I can dig into mods and find it if you think it will help, doesn't sound like it will but I'm happy to help.

**Comment by AlexIIL** [comment]
No it's fine - I already know the cause of this bug (or at least I think I do - I'll push a test release of LBA soon which should fix this.

**Comment by izzmo-tech** [comment]
Hello,
is a solution in sight for this bug? Or can we do anything to make the workbench working again, ie downgrade something?

**Comment by AlexIIL** [comment]
Sorry for the delay - updating LBA to [0.8.6](https://alexiil.uk/maven/alexiil/mc/lib/libblockattributes-all/0.8.6/libblockattributes-all-0.8.6.jar) should fix this.

**Comment by izzmo-tech** [comment]
No problem. Thank you very much, it´s working.
